material.css

```diff
@@ -282,6 +282,7 @@ input.Button[type="submit"] {
 .NotificationsDropdown .Dropdown-toggle {
     width: 36px;
     text-align: center;
+    padding: 0px 0;
```

Comment:
This is working perfect, also fixed the flags button! Thanks!

Reply:
Yep